ansible/roles/operator-pipeline/templates/openshift/tasks/set-cert-project-repository.yml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: set-cert-project-repository
+spec:
+  params:
+    - name: pipeline_image
```

Comment:
Can you please add a description to all parameters?

Reply:
Done